com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl

```diff
@@ -852,11 +852,11 @@ void CompositeOver(real3 colorFront, real3 alphaFront,
 // Space transformations
 // ----------------------------------------------------------------------------
 
-static const float3x3 k_identity3x3 = {1, 0, 0,
+static const float3x3 k_Identity3x3 = {1, 0, 0,
```

Comment:
so I know this is not great but we can't rename it without breaking existing code of users (this is also used by URP).

Here there is not a lot of choice, either we don't rename it, or we do the following.
in common.hlsl:

static const float3x3 k_Identity3x3 = {

(i.e your change).

And in CommonDeprecated.hlsl
we add :
// This is obsolete, don't used, keep for compatiblity.
static const float3x3 k_identity3x3 = k_Identity3x3 ;

this will work and not break existing code.

Reply:
Note that this is a fix. I actually couldn't find it this constant because it was capitalized in the wrong way.
I can fix URP (or already have). I propose we just add a comment in this file. If someone's code breaks, it is trivial to fix.

Reply:
being it a fix or not isn't the problem. The issue is that it will break users custom shader. The proposal is to allow to fix the nomenclature and still not break existing user code (by putting old version as k_identity3x3 )

>"If someone's code breaks, it is trivial to fix" 
We can't afford this anymore. Any change like this must be added to the upgrade guide if there is no automatic upgrade, and for this little change, I will suggest we don't break code.

And yes URP code should be fix in this case as well. You should consider make a separate PR for this change as it isn't related to improve AABB generation and touch a lot of files.